lib/vmpooler/api.rb

```diff
@@ -5,6 +5,8 @@ def initialize
       config_file = File.expand_path('vmpooler.yaml')
       $config = YAML.load_file(config_file)
 
+      $config[:uptime] = Time.now
```

Comment:
It is a Ruby 'time' object; print it displays the string, operations done on it (as I'm doing) are performed on the integer value and returned as seconds.

http://www.ruby-doc.org/core-2.2.0/Time.html#method-i-2D